website/pages/en/cookbook/subgrounds-quickstart.mdx

```diff
@@ -0,0 +1,95 @@
+
+## Step 2: Import Subgrounds and Set Your API Key
+
+Subgrounds allows interfacing with all subgraph endpoints, including both subgraphs in [Subgraph Studio](https://thegraph.com/studio/apikeys/) as well as subgraphs published to The Graph Network. To begin using subgrounds with The Graph Network, import the library and make sure you have a valid API key. You can generate an API key in Subgraph Studio to use it inside the subgraph’s query URL as shown [here](https://thegraph.com/docs/en/querying/querying-the-graph/#using-the-graph-explorer), or create a free Playgrounds API key. Currently Playgrounds offers 5,000 free queries per month.
+
+```python
+from subgrounds import Subgrounds
+
+# Initiate your Subgrounds client with a Playgrounds API key
+sg = Subgrounds.from_pg_key("PLAYGROUNDS_API_KEY")
+```
+
+In this case, Playgrounds API serves as a proxy, routing queries to The Graph Network and handling query fees.
+
+Once installed and imported, it’s easy to get started! Subgrounds offers an intuitive Pythonic API for building GraphQL queries, thus empowering users to download subgraph data into any Python environment without disrupting their current workflows.
+
+## Step 3: Load a Subgraph
+
+```python
+# OpenSea Seaport Ethereum subgraph by Messari
+deployment_id = "QmQBvtHaTS9MftEWYTSmbbmPqzXtMpgZRidivvEaELSKsc"
+
+# Load the subgraph
+seaport = sg.load_subgraph(
+    f"https://api.playgrounds.network/v1/proxy/deployments/id/{deployment_id}"
+)
+```
+
+Subgrounds is built on top of the Python programming language and it is designed to inherit the key advantages of GraphQL. The `load_subgraph` method makes an introspection query to the subgraph, loads the subgraph schema, and then uses the schema to construct and validate queries before making them. This pre-querying validation ensures that only legitimate queries are executed, reducing unnecessary costs for data retrieval and eliminating the risk of inaccurate data.
+
+To learn more about leveraging schema introspection and how subgrounds automates the process, check out this [blog](https://thegraph.com/blog/subgrounds-data-analytics/) by Christophe "Stopher" Vauclair, Co-founder and CTO at Playgrounds.
+
+## Step 4: Query the Subgraph
+
+Now let’s dive into querying the OpenSea Seaport Ethereum subgraph, and see how subgrounds streamlines the process by eliminating the need for raw GraphQL manipulation:
+
+```python
+# Construct your query
+top_collections = seaport.Query.collections(
+    first=1500,
+    orderBy=seaport.Collection.totalRevenueETH,
+    orderDirection='desc',
+)
+```
+
+Note that with subgrounds, enables going beyond [the limit](https://thegraph.com/docs/en/developing/developer-faqs/#23-is-there-a-limit-to-how-many-objects-the-graph-can-return-per-query) of 1,000 items per collection without any need for manual intervention. By default, subgrounds handles GraphQL query pagination automatically, enabling effortless retrieval of large datasets.
```

Comment:
```suggestion
Note that with subgrounds, enables going beyond [the limit](/developing/developer-faqs/#23-is-there-a-limit-to-how-many-objects-the-graph-can-return-per-query) of 1,000 items per collection without any need for manual intervention. By default, subgrounds handles GraphQL query pagination automatically, enabling effortless retrieval of large datasets.
```

Reply:
The Playgrounds team was okay not integrating these docs updates into production as there is already a quckstart for Subgrounds in the docs.